### ISSUE TITLE
Tealium's utag.js is being blocked by Ghostery even when allowed on the site

### ISSUE THREAD

**Issue by simonbrowning** [ghostery/ghostery-extension#519]
### Description

It seems that as of recently Ghostery will now rewrite Tealium's TMS script utag.js and block utag.sync.js by default and even when this is set to "trust on this site".

![image](https://user-images.githubusercontent.com/4973476/77782983-0615ad80-7050-11ea-8e3b-50d8c8b28972.png)

![image](https://user-images.githubusercontent.com/4973476/77783017-129a0600-7050-11ea-9b5f-2f1b5373efa2.png)

### Expected Behavior 

As Tealium's TMS is not tracking users and is a pure TMS, it should not be blocked by default.  Additionally, companies use Tealium's TMS to provide their consent management via this script.

### Actual Behavior 

utag.js and utag.sync.js are being rewritten (307)/blocked on sites unless Ghostery is completely paused on the site.

### Steps to Reproduce

Open new Chrome tab
Open dev tools
Goto vodafone.co.uk
You will see that Ghostery has blocked both utag.sync.js & utag.js

(also see screenshots)

### Versions

+ Browser: Chrome
+ OS: Mac OS
+ Node:
+ NPM:

**Comment by christophertino** [comment]
I'm not seeing utag.sync.js on that site even with Ghostery paused.  However, I do see that the Ad Blocker is not respecting the site-specific whitelisting of Tealium. When I turn off the ad-blocker (blue stop sign) I see Tealium load. I'll dig into that whitelisting bug to see what's happening.

**Comment by rsabec** [comment]
This issue is still occurring in Ghostery version 8.5.1 - created new issue #562 - These files are routinely used as a dependency on site and will cause undefined errors on the page.
